### DISCUSSION TITLE
5.4.1

### DISCUSSION THREAD

**Discussion by kobylynskyi** [kobylynskyi/graphql-java-codegen#981] [🚀 Releases]
## What's Changed
* Do not put validation annotation if return type is a nullable list having non-null values #959 by @kobylynskyi in https://github.com/kobylynskyi/graphql-java-codegen/pull/960
* Fix generated interfaces with disabled `generateParameterizedFieldsResolvers` #976 by @kobylynskyi in https://github.com/kobylynskyi/graphql-java-codegen/pull/977
* Fix invalid parameter names for `extensions` and `customTypeMappings` #964 by @kobylynskyi in https://github.com/kobylynskyi/graphql-java-codegen/pull/979
* Upgrade jackson libs to 2.13.3 #970 by @kobylynskyi in https://github.com/kobylynskyi/graphql-java-codegen/pull/978

**Full Changelog**: https://github.com/kobylynskyi/graphql-java-codegen/compare/v5.4.0...v5.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v5.4.1'>5.4.1</a>.</em>

**Comment by kobylynskyi** [comment]
@jxnu-liguobin could you please release SBT plugin? thanks

**Reply by jxnu-liguobin** [reply]
done
